Signal/src/ViewControllers/ConversationView/Cells/OWSAudioMessageView.m

```diff
@@ -118,7 +121,7 @@ - (void)updateAudioProgressView
     [self.audioProgressView
         setProgress:(self.audioDurationSeconds > 0 ? self.audioProgressSeconds / self.audioDurationSeconds : 0.f)];
 
-    UIColor *progressColor = (self.isIncoming ? [UIColor ows_light02Color] : [UIColor ows_light60Color]);
+    UIColor *progressColor = [self.conversationStyle bubbleSecondaryTextColorWithIsIncoming:self.isIncoming];
```

Comment:
I've tried to pull "dynamic" constants out of views and gather them in the conversation styles, using semantic names.  This will facilitate future work.

Reply:
👍